calibration/DOSTAM/CGINS-DOSTAM-00569__20170829.csv

```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
 569,CC_calibration_date,1503964800000,sensor calibration date (milliseconds since 1970-01-01)
-569,CC_ref_salinity,0,Optode salinity setting
+569,CC_ref_salinity,35,Optode salinity setting
```

Comment:
my assumption is that since the millisecs are matching the filename, that this is the regular case in which we are editing from 0 to 35. If my understanding is correct then this is ok